flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/RemoteInputChannel.java

```diff
@@ -327,6 +322,14 @@ public void notifyBufferAvailable(int numAvailableBuffers) {
 		}
 	}
 
+	@Override
+	public void onCheckpointBarrier(CheckpointBarrier barrier) {
```

Comment:
make it only private method inside `RemoteInputChannel`, because it is never used outside.